android/src/main/java/com/swmansion/rnscreens/ScreenContainer.kt

```diff
@@ -284,11 +271,9 @@ open class ScreenContainer<T : ScreenFragment>(context: Context?) : ViewGroup(co
 
     override fun onMeasure(widthMeasureSpec: Int, heightMeasureSpec: Int) {
         super.onMeasure(widthMeasureSpec, heightMeasureSpec)
-        var i = 0
         val size = childCount
-        while (i < size) {
+        for (i in 0 until size) {
```

Comment:
Is `size` necessary here? I think you could use `childCount` directly.

Reply:
[7aad6e6](https://github.com/software-mansion/react-native-screens/pull/1558/commits/7aad6e6a32178303c9905760644a2cd748911077)